dalek/cleaners/subscriptions.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 var SubscriptionCleaners = []SubscriptionCleaner{
-	deleteNetAppSubscriptionCleaner{},
+	// deleteNetAppSubscriptionCleaner{},
```

Comment:
Can we add a comment showing the error, for us in the future coming back to this?

Reply:
Done 👍